even-digits/go.mod

```diff
@@ -0,0 +1,3 @@
+module even-digits
```

Comment:
Why did you initialize modules in every program directory? There should be a single module (a single `go.mod` file) for all programs.